Base/usr/share/man/man1/yes.md

```diff
@@ -0,0 +1,26 @@
+## Name
+
+yes - output a string repeatedly until killed
+
+## Synopsis
+
+```**sh
+$ yes [string]
+```
+
+## Description
+
+`yes` outputs an endless stream of specified `string` in a loop.
+
+## Options
+
+* `string`: String to print; defaults to "yes".
+
+## Examples
+
+```sh
+$ yes
+yesyesyesyesyesyesyes^C
```

Comment:
`yes` actually prints this:

```
yes
yes
yes
[...]
```

Reply:
It does that in GNU coreutils. Serenity's implementation doesn't add the newline. See [source code](https://github.com/SerenityOS/serenity/blob/master/Userland/Utilities/yes.cpp#L25).

Reply:
`man 3 puts` says this for me:

```
       puts() writes the string s and a trailing newline to stdout.
```

Which matches the behavior I observed when running our `yes` tool.

Reply:
Ok, I fixed that.